is_all_unique_elements.js

```diff
@@ -0,0 +1,16 @@
+
+const isAllUniqueElements = (array) => {
+  let isAllUnique = true;
+
+  array.forEach((currentElement, currentIndex) => {
+    array.slice(currentIndex + 1).forEach((elementToCompare) => {
```

Comment:
Could we change this into a hash lookup to decrease complexity? If we create a hash table of the elements and compare each element to the table, we would still need to loop through the array twice, but could avoid a nested loop.